server.js

```diff
@@ -58,7 +57,7 @@ io.on('connection', function (socket) {
   socket.on('message', function (channel, message) {
     var poll = app.locals.polls[message.id];
     if (channel === 'voteCast') {
-      poll.votes.push(message);
+      poll.responses[socket.id] = message;
       socket.emit('voteCount', poll.countVotes());
       socket.emit('myVoteCast', 'You voted for "' + message + '"');
```

Comment:
@rrgayhart I'm sure it's in this section but haven't had luck dissecting it further. The poll.countVotes() is setup almost the same as the LP